Documentation/dev/libvirt-howto.md

```diff
@@ -81,7 +81,7 @@ echo server=/tt.testing/192.168.124.1 | sudo tee /etc/NetworkManager/dnsmasq.d/t
 ```sh
 go get github.com/crawford/terraform-provider-libvirt
 mkdir -p ~/.terraform.d/plugins
```

Comment:
You can drop these two lines with the `GOBIN` approach.  Test locally with:

```console
$ mv ~/.terraform.d /tmp/  # stick your existing directory somewhere else
$ GOBIN=~/.terraform.d/plugins go get github.com/crawford/terraform-provider-libvirt
$ tree .terraform.d  # see that it worked
.terraform.d
└── plugins
    └── terraform-provider-libvirt

1 directory, 1 file
$ rm -rf ~/.terraform.d  # blow away the test
$ mv /tmp/.terraform.d/ ~  # move the original stuff back into place
```

Reply:
I was too lazy to test this, thanks! Even better now ;-)